prepline_general/api/general.py

```diff
@@ -481,7 +481,7 @@ def return_content_type(filename):
 
 
 @router.post("/general/v0/general")
-@router.post("/general/v0.0.35/general")
+@router.post("/general/v0.0.35-dev0/general")
```

Comment:
okay to use `0.0.35` because we only publish non dev version to dev/prod API, so no need to change this here (and `preprocessing-pipeline-family.yaml`. We do keep a dev version in Changelog just to keep track which one we want to publish :)

Reply:
hm ok. I get the structure of when to use `-dev` and releases for `unstructured`, but is that process different for the API? when do we use dev for the api, and as per your comment is it only added to the changelog (but then the version would be out of sync with `preprocessing-pipeline-family.yaml`?

Reply:
yeah changlog is just for our purpose to keep track which one we should deploy to API and which one we can wait